contrib/debian/changelog

```diff
@@ -1,3 +1,9 @@
+zcash (5.6.1) stable; urgency=medium
```

Comment:
I continue to think that hotfix releases should have `urgency=high` by default. Non-blocking, since that is a complaint about the release script, not this PR.